staging/src/k8s.io/kubectl/pkg/cmd/events/event_printer.go

```diff
@@ -72,10 +73,11 @@ func (ep *EventPrinter) printOneEvent(w io.Writer, e corev1.Event) {
 	}
 	fmt.Fprintf(w, "%s\t%s\t%s\t%s/%s\t%v\n",
 		interval,
-		e.Type,
-		e.Reason,
-		e.InvolvedObject.Kind, e.InvolvedObject.Name,
-		strings.TrimSpace(e.Message),
+		printers.EscapeTerminal(e.Type),
```

Comment:
Aren't events only generated by kubernetes itself?, how user can generate events needed to be escaped?

Reply:
See the `kubectl create` command mentioned in https://github.com/kubernetes/kubernetes/issues/101695#issue-873925774 -- it's possible for a user (with enough permission against say a namespace) to do this, which could then be viewed by an admin or other user.

It might also be possible to get a `\x1B` into an error message that an operator or other component ends up putting into an event -- it doesn't make sense that every component should have to consider terminal escaping that is only relevant to kubectl (in the same way HTML escaping is usually done by the component that renders the HTML).